workers/unity/Assets/Playground/Resources/Prefabs/Common/MobileConnectionSplashScreen.prefab

```diff
@@ -1,970 +0,0 @@
-%YAML 1.1
```

Comment:
Uhhh intentional?

Reply:
yes

Jess removed the screen in a different PR but forgot to remove the related prefab